src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -187,8 +187,10 @@ abstract class FixupSession(
     val future = group.show(range)
     // Make sure the lens is visible.
     ApplicationManager.getApplication().invokeLater {
-      val logicalPosition = LogicalPosition(range.start.line, range.start.character)
-      editor.scrollingModel.scrollTo(logicalPosition, ScrollType.CENTER)
+      if (!editor.isDisposed) {
```

Comment:
unrel fix

---

The error appears when closing the file that contains the content being generated. This fixes it.